src/api/api.js

```diff
@@ -96,7 +96,7 @@ export const apps = [
     "popularity": 2,
     "status": 2,
     "outlook": 3,
-    "description": "After a security breach, Google announced that it would be shutting down its social netowrk Google Plus [in August of 2019(https://www.theverge.com/2018/10/8/17951890/google-plus-shut-down-security-api-change-gmail-android/). The platform never really gained traction outside of niches and was widely regarded as a bad thing when Google rolled it into YouTube's comment system.",
+    "description": "After a security breach, Google announced that it would be shutting down the consumer end of its social netowrk Google Plus in August of 2019. The platform never really gained traction outside of niches and was widely regarded as a bad thing when Google rolled it into YouTube's comment system. The platform will continue to be used in enterprise. Sources: [The Verge](https://www.theverge.com/2018/10/8/17951890/google-plus-shut-down-security-api-change-gmail-android/), [Google](https://www.blog.google/technology/safety-security/project-strobe/)",
```

Comment:
Sorry to nit-pick, but G+ was a social-network in the vain of Facebook. It was integrated with so many of their consumer-facing products. "Moving to enterprise" could be real, or it could be a way to save face. At any rate, it largely fails to be a social network as some sort of private enterprise service. I would call this dead and make another entry for "Plus Enterprise" when that product is launched.

Reply:
That's fair. I'll update accordingly, and add some humor along the way.